packages/plugin-ext-vscode/src/node/plugin-reader.ts

```diff
@@ -0,0 +1,46 @@
+/********************************************************************************
+ * Copyright (C) 2022 Arm
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import { injectable } from '@theia/core/shared/inversify';
+import { Application } from '@theia/core/shared/express';
+import { BackendApplicationContribution } from '@theia/core/lib/node/backend-application';
+
+@injectable()
+export class HostedPluginReader implements BackendApplicationContribution {
+
+    configure(app: Application): void {
+        app.get('/context/:path(*)', async (req, res) => {
```

Comment:
This sets up a single route to serve the web extension init script. Is there a better way?

Reply:
I'll check it out 👍